packages/teams-js/src/public/runtime.ts

```diff
@@ -177,7 +177,7 @@ export let runtime: Runtime | UninitializedRuntime = _uninitializedRuntime;
 
 export const teamsRuntimeConfig: Runtime = {
   apiVersion: 2,
-  hostVersionsInfo: undefined,
+  hostVersionsInfo: { adaptiveCardSchemaVersion: minAdaptiveCardVersion },
```

Comment:
We should probably make a new const for this rather than re-use minAdaptiveCardVersion. What you have here will work, but I'm worried that if we bump the min version in the future we'll accidentally also bump the version that Teams1 is using accidentally.

Reply:
Updated in the latest commit to use new const variable.